mock/handlers.ts

```diff
@@ -0,0 +1,27 @@
+import { forespørselOmDelingAvCvMock } from './foresporsel-om-deling-av-cv-api/mock';
+import { kandidatApiMock } from './kandidat-api/mock';
+import { kandidatsøkMock } from './kandidatsok-proxy/mock';
+import { innloggetBrukerMock } from './meg/mock';
+import { modiaContextHolderMock } from './modiacontextholder/mock';
+import { presenterteKandidaterApiMock } from './presenterte-kandidater-api/mock';
+import { smsApiMock } from './sms-api/mock';
+import { statistikkApiMock } from './statistikk-api/mock';
+import { stillingApiMock } from './stilling-api/mock';
+import { stillingssøkMock } from './stillingssok-proxy/mock';
+import { synlighetApiMock } from './synlighet-api/mock';
+
+const mswHandlers = [
+    ...innloggetBrukerMock,
+    ...kandidatApiMock,
+    ...statistikkApiMock,
+    ...forespørselOmDelingAvCvMock,
+    ...stillingApiMock,
+    ...presenterteKandidaterApiMock,
+    ...smsApiMock,
+    ...synlighetApiMock,
+    ...(import.meta.env.VITE_MOCK_MODIA ? modiaContextHolderMock : []),
```

Comment:
Fant ut at vi alltid bør mocke Modia context holder. `VITE_MOCK_MODIA` styrer om vi mocker selve dekoratøren, ikke kallene som går fra den :) Så kan endre til dette:

```suggestion
    ...modiaContextHolderMock,
```